packages/core/stories/elements/Button/Button.stories.mdx

```diff
@@ -12,160 +12,153 @@ import * as stories from './Button.stories.js';
 
-**Style:**
-The outline button provides a less prominent visual to indicate an less important action on a page, e.g. the Back button. The hover state of the secondary button inverts the background/foreground colors.
 <Canvas>
   <Story
     name="button-secondary-c-primary"
-    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" text="button"/>) } }}>
-      <Button usage="secondary" text="button" />
+    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" text="Secondary"/>) } }}>
+      <Button usage="secondary" text="Secondary" />
   </Story>
   <Story
     name="button-secondary-c-primary-alt"
-    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-primary-alt" text="button"/>) } }}>
-      <Button usage="secondary" theme="c-primary-alt" text="button"/>
+    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-primary-alt" text="Secondary"/>) } }}>
+      <Button usage="secondary" theme="c-primary-alt" text="Secondary"/>
   </Story>
   <Story
     name="button-secondary-c-highlight"
-    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-highlight" text="button"/>) } }}>
-      <Button usage="secondary" theme="c-highlight" text="button"/>
+    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-highlight" text="Secondary"/>) } }}>
+      <Button usage="secondary" theme="c-highlight" text="Secondary"/>
   </Story>
   <Story
     name="button-secondary-c-gray"
-    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-gray-dark" text="button"/>) } }}>
-      <Button usage="secondary" theme="c-gray-dark" text="button"/>
+    parameters={{ docs: { transformSource: () => renderReactMarkup(<Button usage="secondary" theme="c-gray-dark" text="Secondary"/>) } }}>
+      <Button usage="secondary" theme="c-gray-dark" text="Secondary"/>
   </Story>
   <Story story={stories.buttonCSS2} />
 </Canvas>
 
-<hr className="sp--top" />
+---
 
-### Tertiary Button
-**Usage:** For the third action on a page.
+### Tertiary Buttons
 
-**Style:** This is a solid gray button with a lighter background color than the foreground text. The muted color makes this button less prominent on a page than the secondary outline button a providing an option for a tertiary action. The hover state of this button will invert background color with white color text.
+- **Usage:** For third-level actions on a page.
+- **Style:** The gray color is even less prominent than outline button to de-emphasize third-level tasks. The hover state inverts the background and text colors.
```

Comment:
```suggestion
- **Style:** The gray color is even less prominent than outline button to de-emphasize third-level tasks. Background color of the primary button lightens when being hovered over.
```

Sorry, these are the things I missed in the first draft.

Reply:
Ah, right. I would maybe word it a bit differently:
"The hover state lightens the primary button background color."

Reply:
oh sorry, this is not the primary button, but the hover animation is similar to the primary button, so I just copied from the primary. This is actually the tertiary button